plenario/models.py

```diff
@@ -464,6 +467,10 @@ def index(cls, geom=None):
 
         return listing
 
+    @classmethod
```

Comment:
Why is this a class method? Looks like we don't use `cls`. Unless you wanted to sub in `cls` for `ShapeMetadata` in `ShapeMetadata.dataset_name`